webpack.config.js

```diff
@@ -78,13 +77,9 @@ module.exports = {
   optimization: {
     minimize: isProduction,
     minimizer: [
-      new TerserPlugin({
-        extractComments: false,
-        terserOptions: {
-          output: {
-            comments: false,
-          },
-        },
+      new ESBuildMinifyPlugin({
+        target: 'es2015', // Syntax to compile to (see options below for possible values)
```

Comment:
```suggestion
        target: 'es2015',
```

Reply:
^

(This comment was copied from the module README, should not be there)